pkg/cgroups/cgroups_unsupported.go

```diff
@@ -4,7 +4,10 @@
 package cgroups
 
 import (
+	"fmt"
 	"os"
+
+	systemdDbus "github.com/coreos/go-systemd/v22/dbus"
```

Comment:
Does this library compile on non-linux? Is there a way to fake the prototype so it isn't necessary to pull in this package?

Reply:
that is a good idea, I'll look into it.  For now I've just tried to fix some of the issues I've introduced with https://github.com/containers/common/pull/1716 that prevented revendoring into Podman: https://github.com/containers/podman/pull/20568

There is still room for cleanups in the cgroups package.